interface/src/app/heatmap/heatmap-vgspec.js

```diff
@@ -80,8 +98,8 @@ angular.module('adage.heatmap-vgspec', [])
       'name': 'samples',
       'type': 'ordinal',
       'domain': {
-        'data': 'activity_normalized',
-        'field': 'sample_order._id',
+        'data': 'samples_labeled',
```

Comment:
I noticed that you changed "sample_objects" to "sampleObjects" earlier. If you are trying to convert the variables to camelCase, you may want to convert this one as well.

Reply:
Aha... this is inside the Vega spec, which is actually JSON, not JavaScript. I had switched the `sampleObjects` label since that dataset originated from the JavaScript that invokes the heatmap. I took a quick pass through the examples on the Vega web site and they are all over the map with camelCase, snake_case, and sometimes even both in the same spec.

It looks as if there isn't really a more general standard for JSON either. StackOverflow has this good discussion, which pretty much covers all of the angles:

> https://stackoverflow.com/questions/5543490/json-naming-convention

I think we should lean toward camelCase, since we're using AngularJS (JavaScript, obviously) as a supporting framework. I will change just this label now to minimize last minute code changes and do the rest as part of our migration from Vega 2 to Vega 3.